File: c/SumOfSubArrays.c
#include<stdio.h>

int main(){
    int arr[] = {1,2,3,4,5,6};
    int n = 6;

    for(int i=0;i<n;i++){
        int sum =0;
        for(int j=i;j<n;j++){
            sum+=arr[j];
        }
        print("%d",sum);        
    }

    return 0;
}

File: python/SumOfSubArrays.py
def sum(arr,n):

    for i in range(0,n):
        res=0
        for j in range(i,n):
            res+=arr[j]

        print(res, end = ' ')
    
arr[] = [1,2,3,4,5,6]
sum(arr,len(arr))
